src/test/ui/lint/clashing-extern-fn.rs

```diff
@@ -233,7 +236,7 @@ mod missing_return_type {
             // the correct one -- if this one is the correct one, then calling the usize-returning
             // version would allow reads into uninitialised memory.
             fn missing_return_type();
-            //~^ WARN `missing_return_type` redeclared with a different signature
+        //~^ WARN `missing_return_type` redeclared with a different signature
```

Comment:
there are a bunch of somewhat weird identation changes here, were these intentional?

Reply:
Oops, no, these were from rustfmt